app/_hub/kong-inc/forward-proxy/index.md

```diff
@@ -108,6 +108,10 @@ params:
       description: |
         The username to authenticate with, if the forward proxy is protected
         by basic authentication.
+
+        This field is _referenceable_, which means it can be securely stored as a
+        [secret](/gateway/latest/plan-and-deploy/security/secrets-management/getting-started)
+        in a vault and called with a [reference](/gateway/latest/plan-and-deploy/security/secrets-management/reference-format).
```

Comment:
I'm not sure if `... and called with a reference` is the right wording here.

`This field is _referenceable_, which means it can be securely stored as a secret in a vault. References follow a specific format -> link`

or 

`This field is _referenceable_, which means it can be securely stored as a secret in a vault. You can use references->link to a secret`

or

`This field is _referenceable_, which means it can be securely stored as a secret in a vault. How to setup references to a secret can be found here -> link`

WDYT @lena-larionova ?

Reply:
I'm good with the first one.